src/main.py

```diff
@@ -86,8 +83,8 @@ def on_draw(self):
 
         # clear screen
         self.clear()
-        self.all_sprites.draw()
         self.sprite_lists.draw()
+        self.all_sprites.draw()
```

Comment:
We should totally eliminate `all_sprites`, right?  I can make a separate github issue to track.

Reply:
Fully agree. This could be a task in Trello.

Reply:
Awesome, I've created a card.